vagrant-spk

```diff
@@ -511,6 +511,14 @@ def setup_vm(args):
     with open(vagrantfile_path, "w") as f:
         f.write(VAGRANTFILE_CONTENTS)
 
+    # Recursively copy in anything in the stack's service-config folder.
```

Comment:
But then that's code complexity that we have to maintain.  I assert that the maintenance cost of this duplication is lower than the cost of implementing one of the alternatives.